runtime/lua/vim/lsp/buf.lua

```diff
@@ -40,8 +40,12 @@ end
 
 --- Displays hover information about the symbol under the cursor in a floating
 --- window. Calling the function twice will jump into the floating window.
-function M.hover()
-  local params = util.make_position_params()
+---@param options table|nil additional options
```

Comment:
```suggestion
---@param opts? table additional options
```
Not only this place.

Also whitespace is not consistent on `col`.

Reply:
I originally named this as `opts`, but wanted consistency with the arguments that other members of this module already have. Should everything be `opts?`?

Reply:
I see. Options would be fine then. Either we change all or remain consistent.

Reply:
> Should everything be `opts?`?

in favor of that